File: URI1036.cpp
#include <iostream>
#include <cstdio>
#include <math.h>

using namespace std;

int main() {
    double A, B, C, delta , R1, R2;

    cin >> A >> B >> C;

    delta = ((B*B) - 4*(A)*(C));
    
    if (A == 0.00 || delta < 0.00){
        printf("Impossivel calcular\n");
    }
    else {
        if (delta > 0) {
            R1 = (-B + (sqrt(delta))) / (2 * A);
            R2 = (-B - (sqrt(delta))) / (2 * A);

            printf("R1 = %.5f\n", R1);
            printf("R2 = %.5f\n", R2);
        }
    }

    return 0;
}

File: 1. Beginner/URI1036.py
from math import sqrt

a,b,c = input().split(" ")

A = float(a)
B = float(b)
C = float(c)

delta = ((B*B) - 4*(A)*(C))


if A  == 0 or delta < 0:
    print("Impossivel calcular")
else:
    if delta > 0:
        R1 = (-B + (sqrt(delta))) / (2 * A)
        R2 = (-B - (sqrt(delta))) / (2 * A)

        print("R1 = %.5f" %R1)
        print("R2 = %.5f" %R2)
